.github/workflows/deploy-pip-poetry.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy
+
+on:
+
+  workflow_call:
+    inputs:
+        os:
+          required: true
+          type: string
+        python-version:
+          required: true
+          type: string
+        publish:
```

Comment:
The `publish` input is not used in the workflow

Reply:
I see. If it is not necessary I will remove it.